lib/modules/manager/bazel/rules/go.ts

```diff
@@ -29,10 +29,10 @@ export const GoTarget = z
       }
 
       if (commit) {
-        dep.currentValue = 'v0.0.0';
         dep.currentDigest = commit;
-        dep.currentDigestShort = commit.substring(0, 7);
-        dep.digestOneAndOnly = true;
+        if (!tag) {
+          dep.digestOneAndOnly = true;
```

Comment:
🤔 where is that property used? never seen it before

Reply:
Only with `go`, and actually I'm not sure what it's for exactly

Reply:
It was a hack when I added Go. I forget exactly why or if it's still necessary

Reply:
I saw no difference with or without it

Reply:
Let's keep it for now